internal/app/engine/config.go

```diff
@@ -54,6 +54,7 @@ var defaultTeams = []string{
 	"URoboRus",
 	"Warthog Robotics",
 	"ZJUNlict",
+	"DelftMercurians",
```

Comment:
The team names are sorted alphabetically. Please insert your team name accordingly.